### PR TITLE
WIP: Display cancelled vehicule in green background with red cross

### PR DESCRIPTION
**Pull request by edellucien**
Display cancelled vehicule in green background with red cross, on 'useDelayStyle' mode

Ticket: TRAFDIV-361
# How to

# Others

- [ ] It's not a hack or at least an unauthorized hack :).
- [ ] The images added are optimized.
- [x] Everything in ticket description has been fixed.
- [x] The author of the MR has made its own review before assigning the reviewer.
- [x] The title means something for a human being.
- [x] The title contains [WIP] if it's necessary.
- [x] Labels applied. if it's a release? a hotfix?
- [ ] The new class' members & methods are well documented
- [ ] Tests added.

### CODE REVIEW

File: src/common/trackerConfig.js [hunk 1]
```diff
@@ -156,9 +156,9 @@ export const getTextSize = (ctx, markerSize, text, fontSize) => {
 /**
  * @ignore
  */
-export const getDelayColor = (delayInMs, cancelled) => {
+export const getDelayColor = (delayInMs, cancelled, isText) => {
```
Comment (mhaertwig) [inline src/common/trackerConfig.js:159]: What does `isText` mean?

Comment (edellucien) [reply src/common/trackerConfig.js:159]: It is when we want to retrieve the color for the delay text, not the background.

Comment (oterral) [reply src/common/trackerConfig.js:159]: Describe the parameters of the functions.

**Review by oterral — CHANGES_REQUESTED** [review]
Add a description of the parameters of the function

**Review by oterral — APPROVED** [review]

**Pull request merged**